examples/cloud_subnet/main.tf

```diff
@@ -47,3 +47,11 @@ resource "aci_cloud_subnet" "cloud_subnet" {
   usage              = "gateway"
   zone               = "uni/clouddomp/provp-aws/region-us-west-1/zone-us-west-1b"
 }
+
+resource "aci_cloud_subnet" "gcp_cloud_subnet" {
+  cloud_cidr_pool_dn = aci_cloud_cidr_pool.cloud_cidr_pool.id
+  ip                 = "10.0.1.0/24"
+  usage              = "gateway"
+  zone               = "uni/clouddomp/provp-gcp/region-asia-east1/zone-default"
```

Comment:
I think we should use the datasource here like: data.aci_cloud_availability_zone.az_us_east_1_aws.id